.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
```

Comment:
As a general comment: if this cannot be AGS wide requirements, at least, this needs to be reviewed and adopted in mathlib organization.

Reply:
We can propagate this to the org once we've used it and iterated on it. Let's revisit this later.